tutorials/cuda_mps/README.md

```diff
@@ -185,6 +205,83 @@ We vary the number of parallel inferences to show that MPS may not be beneficial
 
 As the number of parallel inferences increases, so does the workload, and the benefit of MPS is more evident. However, when the workload is low, CUDA MPS may not be beneficial. 
 
-| Maximum Latency for 5 Instances | 
+| Maximum Latency for 5 Instances |
+| :-------------------------:|
+![max latency for 5 parallel inf](images/igx-dgpu/parallel_inf_Maximum%20Latency%20(ms).png)|
+
+###  IGX Orin w/ Integrated GPU
+#### MPS Setup on IGX-iGPU
+> Note that we run all commands as root
+
+**1. Please add cuda-12.5+ to `$PATH` and `$LD_LIBRARY_PATH`**
+
+```bash
+echo $PATH
+/usr/local/cuda-12.6/compat:/usr/local/cuda-12.6/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
+
+echo $LD_LIBRARY_PATH
+/usr/local/cuda-12.6/compat/lib:/usr/local/cuda-12.6/compat:/usr/local/cuda-12.6/lib64:
+```
+
+**2. Be sure to pass `-v /tmp/nvidia-mps:/tmp/nvidia-mps  -v /tmp/nvidia-log:/tmp/nvidia-log -v /usr/local/cuda-12.6:/usr/local/cuda-12.6 ` to `docker run` to ensure that the container is connected to the MPS control and server**
+
+Example:
```

Comment:
Could you add a note on how the lengthy `docker run` command below was generated? Probably `dev_container launch --img <img> --dryrun` or similar?

Reply:
Fixed it.